ios/browser/api/ntp_background_images/ntp_sponsored_image+private.h

```diff
@@ -30,7 +30,8 @@ struct TopSite;
 
 @interface NTPSponsoredImageBackground (Private)
 - (instancetype)initWithSponsoredBackground:
-    (const ntp_background_images::Creative&)sponsoredBackground;
+                    (const ntp_background_images::Creative&)sponsoredBackground
```

Comment:
I have moved `shouldMetricsFallbackToP3A` to Creative, so that you do not need to pass a new param.